### DISCUSSION TITLE
Following Remote Cursor Across Monitors

### DISCUSSION THREAD

**Discussion by Acivev** [rustdesk/rustdesk#7906] [General]
While a multi-monitor setup can enhance productivity and simplify tasks, manually switching between screens can be tedious. I hope this feature request can address that. :3

![monitors](https://github.com/rustdesk/rustdesk/assets/37246611/80022df8-564b-45ed-9f22-a5a650905ba4)

place of settings to enable follow remote cursor
![enable_follow_cursor](https://github.com/rustdesk/rustdesk/assets/37246611/7d1f94ca-96a0-402c-b473-8c7f98776a48)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/7717
